torch/csrc/fx/node.cpp

```diff
@@ -1,11 +1,14 @@
 #include <torch/csrc/fx/node.h>
 
+#include <c10/util/SmallVector.h>
 #include <structmember.h>
 #include <torch/csrc/utils/object_ptr.h>
 #include <torch/csrc/utils/pythoncapi_compat.h>
+#include <algorithm>
 
 namespace {
 
+typedef c10::SmallVector<int64_t, 2> NodeSortKey;
```

Comment:
```suggestion
using NodeSortKey = c10::SmallVector<int64_t, 2>;
```

Curious about the 2 here. I think clang implements std::vector as smallvector 4, with for 4 stack allocations. Is there any benefit to using something smaller like 2?

Reply:
This list length is 1 for newly constructed, 2 if you insert in the middle, so I figured that was the common case,  I'll increase for 4.